tool-execution/run_pmd.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+## This script sets up, and runs PMD on an Ubuntu 20.04 or MacOS 11 machine.
+###########
+## SETUP ##
+###########
+## checks if the environment variable pmd is set
+if ! command -v pmd &> /dev/null ; then
+    echo "cannot find pmd on this machine."
+    echo "run the script tool-execution/pmd-setup.sh if you are on an Ubuntu 20.04 machine or Mac OS 11+. Otherwise, read the instructions in that script on how to adapt it for other platforms, then follow them for yours."
+    ## run pmd-setup.sh to setup pmd
+    ./pmd_setup.sh
```

Comment:
Running this script here contradicts the instructions in the `echo` on the prior line. Should this script do it, or should a human?